scripts/all-tests.sh

```diff
@@ -15,27 +15,27 @@ set -e
 #   Possible values are:
 #
 #       - "quality": Run the quality (pep8/pylint) checks
-#       - "unit": Run the JavaScript and Python unit tests
-#            (also tests building the Sphinx documentation,
-#             because we couldn't think of a better place to put it)
+#       - "lms-unit": Run the LMS Python unit tests
+#       - "cms-unit": Run the CMS Python unit tests
+#       - "commonlib-js-unit": Run the JavaScript tests and the Python unit
+#         tests from the common/lib directory
 #       - "lms-acceptance": Run the acceptance (Selenium) tests for the LMS
 #       - "cms-acceptance": Run the acceptance (Selenium) tests for Studio
 #       - "bok-choy": Run acceptance tests that use the bok-choy framework
 #
-#   `SHARD` is a number (1, 2, or 3) indicating which subset of the tests
-#       to build.  Currently, "lms-acceptance" and "bok-choy" each have two
-#       shards (1 and 2), "cms-acceptance" has three shards (1, 2, and 3),
-#       and all the other test suites have one shard.
+#   `SHARD` is a number indicating which subset of the tests to build.
 #
-#       For the "bok-choy", the tests are put into shard groups using the nose
-#       'attr' decorator (e.g. "@attr('shard_1')").  Currently, anything with
-#       the 'shard_1' attribute will run in the first shard.  All other bok-choy
-#       tests will run in shard 2.
+#       For "bok-choy" and "lms-unit", the tests are put into shard groups
+#       using the nose'attr' decorator (e.g. "@attr('shard_1')"). Anything with
+#       the 'shard_n' attribute will run in the nth shard. If there isn't a
+#       shard explicitly assigned, the test will run in the last shard (the one
+#       with the highest number).
 #
 #       For the lettuce acceptance tests, ("lms-" and "cms-acceptance") they
 #       are decorated with "@shard_{}" (e.g. @shard_1 for the first shard).
 #       The lettuce tests must have a shard specified to be run in jenkins,
-#       as there is no shard that runs unspecified tests.
+#       as there is no shard that runs unspecified tests. (NOTE: We no longer
+#       have multiple shards for lettuce tests)
```

Comment:
Since we don't have it any more and haven't for a while, it probably makes sense to take out any reference to shards here.